t/deploy.t

```diff
@@ -18,7 +18,7 @@ use OpenQA::Test::TimeLimit '20';
 use OpenQA::Test::Case;
 use Mojo::File 'path';
 use List::Util 'min';
-use Try::Tiny;
+use Feature::Compat::Try;
```

Comment:
Where is the actual change to the try/catch then?

Reply:
what do you mean `actual change`?

Reply:
Well, you only replaced the import but Try::Tiny and Feature::Compat::Try use different syntax. In the meantime you already changed the code by removing the semicolon but as I commented in https://github.com/os-autoinst/openQA/pull/6183#discussion_r1971094512 you should remove this import and change the code to use Test::Exception functions

Reply:
With actual change I meant, you changed the module, but not the actual `try` statement.
The error message was:
https://app.circleci.com/pipelines/github/os-autoinst/openQA/15882/workflows/87ef8c53-4203-44c1-9a58-398412fdd72f/jobs/152826
```
[17:34:55] t/deploy.t ................................................ Expected a catch {} block at t/deploy.t line 51.
```
That means Feature::Compat::Try always wants a `catch` block.